### PR TITLE
Initial German Translation

### PR DESCRIPTION
**Pull request by alp10711**
Hello,

I have added an initial translation into German :smiley:.

Best regards
Alp

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+#: data/com.vixalien.sticky.desktop.in.in:3
+#: data/com.vixalien.sticky.appdata.xml.in.in:7
+msgid "Pin notes to your desktop"
+msgstr "Notizen an deinen Desktop anheften"
+
+#: data/com.vixalien.sticky.desktop.in.in:4
+#: data/com.vixalien.sticky.appdata.xml.in.in:6 data/ui/notes.ui:32
+#: src/application.ts:68 src/application.ts:359
+msgid "Sticky Notes"
+msgstr "Sticky Notes"
```
Comment (vixalien) [inline po/de.po:30]: Why didn't you translate the name of the app here?

File: po/de.po [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+msgstr "Durchgestrichen"
+
+#: data/ui/window.ui:138
+msgid "All Notes"
+msgstr "Alle Notizen"
+
+#. TRANSLATORS: eg. 'Translator Name <your.email@domain.com>' or 'Translator Name https://website.example'
+#: src/application.ts:371
+msgid "translator-credits"
+msgstr "alp10711 <93343276+alp10711@users.noreply.github.com>"
```
Comment (vixalien) [inline po/de.po:285]: Please use a real email or no email at all

File: po/de.po (hunk 1 above)
Comment (alp10711) [reply po/de.po:30]: Oh, a mistake has crept in there :smile:

File: po/de.po (hunk 2 above)
Comment (alp10711) [reply po/de.po:285]: Okay, I got it

**Pull request merged**